jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamodb/editor/DynamoDbTableEditor.kt

```diff
@@ -114,11 +119,16 @@ class DynamoDbTableEditor(private val dynamoTable: DynamoDbVirtualFile) : UserDa
         coroutineScope.launch(edt) {
             searchResults.setBusy(true)
             val (index, partiqlStatement) = searchPanel.getSearchQuery()
+            val fetchType = when (searchPanel.searchType) {
+                SearchPanel.SearchType.Scan -> DynamoDbFetchType.Scan
+                SearchPanel.SearchType.Query -> DynamoDbFetchType.Query
+            }
 
             withContext(bg) {
                 LOG.debug { "Querying Dynamo with '$partiqlStatement'" }
 
                 val request = ExecuteStatementRequest.builder().statement(partiqlStatement).build()
+                val succeeded = AtomicBoolean(true)
```

Comment:
succeeded is not accessed across threads, so doesn't need atomic

Reply:
yeah you're right - i just hate using `var` - I always feel like I've failed when I do :)

will fix.